backends/rapidpro/msg.go

```diff
@@ -512,15 +512,15 @@ type DBMsg struct {
 }
 
 func (m *DBMsg) ID() courier.MsgID            { return m.ID_ }
-func (m *DBMsg) EventID() int64               { return m.ID_.Int64 }
+func (m *DBMsg) EventID() int64               { return int64(m.ID_) }
 func (m *DBMsg) UUID() courier.MsgUUID        { return m.UUID_ }
 func (m *DBMsg) Text() string                 { return m.Text_ }
 func (m *DBMsg) Attachments() []string        { return []string(m.Attachments_) }
-func (m *DBMsg) ExternalID() string           { return m.ExternalID_.String }
+func (m *DBMsg) ExternalID() string           { return string(m.ExternalID_) }
 func (m *DBMsg) URN() urns.URN                { return m.URN_ }
 func (m *DBMsg) URNAuth() string              { return m.URNAuth_ }
 func (m *DBMsg) ContactName() string          { return m.ContactName_ }
-func (m *DBMsg) HighPriority() bool           { return m.HighPriority_.Valid && m.HighPriority_.Bool }
+func (m *DBMsg) HighPriority() bool           { return m.HighPriority_ }
```

Comment:
This is a side effect of taking out the null library in that we don't have a Bool version in the Nyaruka version. So we will be writing FALSE instead of NULL. That this field is nullable seems broken anyways.

Reply:
Guessing it's that way because it doesn't have a meaning for incoming messages? `FALSE` does seem fine in that case.